File: 75-sort-colors/sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    let t
    for(let i=0; i<nums.length; i++)
        {
            for(let j=0; j<nums.length; j++)
            {
                if(nums[i] > nums[j]){
                t = nums[i]
                nums[i] = nums[j]
                nums[j] = t
            }
            if(nums[j] > nums[i]){
                t = nums[j]
                nums[j] = nums[i]
                nums[i] = t
            }
            }
        }
};

File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zeros=0, ones=0, twos=0;
        for(auto x: nums){
            zeros += x==0;
            ones += x==1;
            twos += x==2;
        }
        int i = 0;
        while(zeros--) nums[i]=0, i++;
        while(ones--) nums[i]=1, i++;
        while(twos--) nums[i]=2, i++;
    }
};